### ISSUE TITLE
Data Science with Python : Object Detection

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#437]
Welcome to 'DSWP' Team, good to see you here

This issue will helps in knowing complete process of how an object is detected using Computer Vision. Tutorial to Computer Vision and how it's applied using sample codes with brief examples.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Computer Vision

Mentors Assigned : Jivitesh Jain / Kareem Negm

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

**Comment by HastiSutaria** [comment]
I would like to work on this.
Hasti Sutaria - 426
Type: Documentation
Approach: Working with OpenCV library for object detection.

**Comment by prathimacode-hub** [comment]
Issue assigned to @HastiSutaria for documentation

**Comment by HastiSutaria** [comment]
@prathimacode-hub Thank you! Object Detection's documentation on me.

**Comment by m0hit-kumar** [comment]
I want to contribute to this issue :
Name-Mohit Kumar
Domain-Data Science with Python
Batch-4
Issue Number-437
Contribution Approaches- how to detect and load images using OpenCV, use it to detect objects from an image

**Comment by shripaddhopate** [comment]
I would be glad to work on this issue.
Name : Shripad Dhopate
Batch : 21
Type : Documentation , Video
Serial No: 930

**Comment by Kareem-negm** [comment]
Hello, the mentor here for any help

**Comment by HastiSutaria** [comment]
Hello, I wanted to know what further steps I need to take to create a pull
request and submit the solved issue task documentation. So, can you please
guide me with it?

‪On Mon, 20 Sept 2021 at 15:15, ‪Karim Negm‬‏‎ ***@***.***>
wrote:‬

**Comment by Kareem-negm** [comment]
Hello @HastiSutaria ,
in this [artical ](https://www.dataschool.io/how-to-contribute-on-github/) you can find how to make a step by step PR 
or you can see this [video](https://www.youtube.com/watch?v=Aj4Bo8-yTT0&feature=youtu.be) 
if you have any question please let me know

**Comment by HastiSutaria** [comment]
@Kareem-negm Under Datascience_With_Python branch "object detection" will come under Computer vision? And Should I upload   the google colab there?

**Comment by Kareem-negm** [comment]
Hello @HastiSutaria  , Yes exactly as you say 
please if you upload the file made a PR and mention this issue

**Comment by HastiSutaria** [comment]
Yes! It's almost done. Will upload soon.

**Comment by HastiSutaria** [comment]
@prathimacode-hub @Kareem-negm I am getting a small error. Can I get help?

**Comment by Kareem-negm** [comment]
Of course, I am here to help you

**Comment by HastiSutaria** [comment]
PFA screenshot. 
![Screenshot 2021-09-26 22 29 17](https://user-images.githubusercontent.com/69644003/134816963-054778e4-dbdd-4d03-b26d-0ad82bc655c5.png)

**Comment by HastiSutaria** [comment]
Please find attached screenshot of the error.
![Screenshot 2021-09-26 22 29 17](https://user-images.githubusercontent.com/69644003/134817084-79d6d26e-a535-48a9-af43-63937ca173f5.png)

**Comment by HastiSutaria** [comment]
Please find attached screenshot @Kareem-negm 
![Screenshot 2021-09-26 223739](https://user-images.githubusercontent.com/69644003/134817191-8be93e8d-e044-4530-b2f4-ecb53d76d3a8.png)

**Comment by Kareem-negm** [comment]
I think it's about OpenCV version

**Comment by HastiSutaria** [comment]
I downloaded latest opencv version.

**Comment by Kareem-negm** [comment]
Yes, I mean maybe this method may be changed in this version, and you must work on the different one 
add me to the notebook to take a look

**Comment by HastiSutaria** [comment]
Have created my PR! @Kareem-negm

**Comment by Kareem-negm** [comment]
I mean on Google Colab 
via my Gmail: `negmkareem38@gmail.com`

**Comment by HastiSutaria** [comment]
@prathimacode-hub Kindly review my PR ma'am.

**Comment by HastiSutaria** [comment]
@prathimacode-hub requested changes completed. PR no #7548
